kdwsdl2cpp/src/main.cpp

```diff
@@ -125,6 +130,13 @@ int main(int argc, char **argv)
             }
         } else if (opt == QLatin1String("-keep-unused-types")) {
             keepUnusedTypes = true;
+        } else if (opt == QLatin1String("-import-path")) {
+            ++arg;
+            if (!argv[arg]) {
+                showHelp(argv[0]);
+                return 1;
+            }
+            importPathList.append(argv[arg]);
```

Comment:
append(QFile::decodeName(argv[arg])), to handle correctly non-ascii characters [on Unix at least...].